Kinvey.Tests/UserUnitTests.cs

```diff
@@ -172,7 +172,8 @@ public async Task TestMICValidateAuthServiceID()
 		}
 
         [TestMethod]
-		public async Task TestMICRenderURLScopeID()
+        [Obsolete("This test has been deprecated.")]
+        public async Task TestMICRenderURLScopeID()
```

Comment:
We are not deprecating the normal MIC flow (Authorization Grant Flow), why is this test method being deprecated?